<!--
    This test can be launched with
    > rostest openzen_sensor openzen_test.launch
-->
<launch>
    <!-- IG1 Sensor node -->
    <node name="openzen_testsensor_node" pkg="openzen_sensor" type="openzen_sensor_node">
        <param name="sensor_interface" type="str" value="TestSensor" />
        <param name="sensor_name" type="str" value="Sensor1" />
    </node>

    <!-- Quaternion to Euler conversion node -->
    <node name="quat_to_euler" pkg="openzen_sensor" type="quat_to_euler_node" />

    <test test-name="test_openzen_node" pkg="openzen_sensor" type="test_openzen_readout.py" />
</launch>
